File: Models/BITACORA.cs
namespace Models
{
    public class BITACORA
    {
        public int IdBitacora { get; set; }
        public int IdTipoMovimiento { get; set; }
        public decimal Monto { get; set; }
        public int IdUsuario { get; set; }
        public DateTime FecRegistro { get; set; }
        public string NombreCompleto { get; set; }
    }
}

File: PIA_PD/ClientApp/src/logic/Bitacora.js
import axios from 'axios';

const ENDPOINT_PATH = 'api/bitacora'
export default {
    listaBitacoras() {
        return axios.get(ENDPOINT_PATH + '/Read');
    },
    agregarBitacora(objeto){
        return axios.post(ENDPOINT_PATH+"/Create", {
            IdTipoMovimiento : objeto.idTipoMovimiento,
            Monto : objeto.monto,
            IdUsuario : localStorage.IdUsuario
        });
    },
    modificarBitacora(objeto){
        return axios.put(ENDPOINT_PATH+"/Update", {
            IdBitacora : objeto.idBitacora,
            Monto : objeto.monto,
        });
    }
}